dc-cudami-admin/dc-cudami-admin-webapp/src/main/java/de/digitalcollections/cudami/admin/controller/identifiable/resource/FileResourcesMetadataController.java

```diff
@@ -118,6 +128,24 @@ public ResponseEntity save(@RequestBody FileResource fileResource)
     }
   }
 
+  @GetMapping("/api/fileresources/images")
+  @ResponseBody
+  public SearchPageResponse<FileResource> search(
+      @RequestParam(name = "nullHandling", required = false, defaultValue = "NATIVE")
```

Comment:
Für was ist denn dieses Parameter notwendig? Das klingt für mich sehr nach einer leaking Abstraction, warum muss sich der Client darum scheren wie wir mit `null` Werten umgehen? Und in welchem Kontext?

Reply:
Dürfte es wohl tatsächlich nicht brauchen.